src/Parser/Core/Combatant.js

```diff
@@ -110,6 +110,8 @@ class Combatant extends Entity {
         return 1.04 + this.masteryRating / 3076.96;
       case SPECS.FURY_WARRIOR:
         return 0.11 + this.masteryRating / 28430;
+      case SPECS.FROST_DEATH_KNIGHT:
+        return 0.12 + this.masteryRating / 266636;
```

Comment:
From my understanding of how mastery works, that divisor is a typo right? Should be `26666` not `266636`?

Reply:
Is that method even still used? Can we remove it? :P

Reply:
It may still be called in some places, but those places should be migrated and yeah it should be removed.

Reply:
Huh didn't know this was out of use.  Fixed the typo anyways